.github/workflows/bot-community-calls.yml

```
@@ -37,6 +37,6 @@ jobs:
             - name: Check Schedule and Notify
               env:
                   GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-                  DRY_RUN: "false" # will post when workflow is triggered by cron
+                  DRY_RUN: ${{ inputs.dry-run }}
```

actionlint (1.7.10)

property "dry-run" is not defined in object type {dry_run: string}

(expression)